src/inc/iceefilegen.h

```diff
@@ -76,6 +76,8 @@ typedef HRESULT (__stdcall * PFN_DestroyICeeFileGen)(ICeeFileGen ** ceeFileGen);
 
 class ICeeFileGen {
   public:
+    virtual ~ICeeFileGen() = default;
```

Comment:
This probably has the same issue as `LLDBServices`, there's probably a class that implements this interface and that class needs a virtual destructor.

Reply:
Ah, I see. This isn't actually a COM interface, despite the name. So I suppose this one is fine.